File: project_euler/Problem025/Problem025.go
package project_euler

import (
    "fmt"
    "math"
)

func Problem025() {
    target := 1000
    result := int(math.Ceil((float64(target) - 1 + math.Log10(5) / 2) / math.Log10((1 + math.Sqrt(5)) / 2)))
    fmt.Println(result)
}

File: project_euler/Problem025/Problem025.java
package project_euler;

public class Problem025 {
    public void solution() {
        int target = 1000;
        int result = (int) Math.ceil(((double) target - 1 + Math.log10(5) / 2) / Math.log10((1 + Math.sqrt(5)) / 2));
        System.out.println(result);
    }
}
